### PR TITLE
Reduce yaml files in folder textmode_installation_minimal_role

### PR DESCRIPTION
**Pull request by tinawang123**
Reduce yaml files in folder schedule/yast/textmode_installation_minimal_role

- Related ticket: https://progress.opensuse.org/issues/153071
- Verification run: https://openqa.suse.de/tests/13276478#
   https://openqa.suse.de/tests/13276479
   https://openqa.suse.de/tests/13276480
- Related MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/38

### CODE REVIEW

**Review by sofiasyria — APPROVED** [review]
LGTM, just add the MR for changing the schedules in the Yast job group

**Comment by jknphy** [comment]
A couple of comments:
- we plan to move installations to test_cases/ folder, that is not done
- YUI_REST_API can also be moved to job group yaml, so in the future we have this settings perhaps shared in job group yaml with some anchor/reference and later on deleted due to all we have is using yui_rest_api.

**Comment by tinawang123** [comment]
> A couple of comments:
> 
>     * we plan to move installations to test_cases/ folder, that is not prepared yet.
> 
>     * YUI_REST_API can also be moved to job group yaml, so in the future we have this settings perhaps shared in job group yaml with some anchor/reference and later on deleted due to all we have is using yui_rest_api.

I have moved 'YUI_REST_API' to the  job group yaml.   For the first comment, should I do anything?

**Comment by jknphy** [comment]
> > A couple of comments:
> > ```
> > * we plan to move installations to test_cases/ folder, that is not prepared yet.
> > 
> > * YUI_REST_API can also be moved to job group yaml, so in the future we have this settings perhaps shared in job group yaml with some anchor/reference and later on deleted due to all we have is using yui_rest_api.
> > ```
> 
> I have moved 'YUI_REST_API' to the job group yaml. For the first comment, should I do anything?

test cases should be stored here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/schedule/yam/test_cases

**Comment by tinawang123** [comment]
> test cases should be stored here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/schedule/yam/test_cases

Moved to 'schedule/yam/test_cases/textmode_installation_minimal_role.yaml

File: schedule/yam/test_cases/textmode_installation_minimal_role.yaml [hunk 1]
```diff
@@ -6,11 +6,9 @@ description:    >
     2. "Minimal" role is selected;
     3. Boot to command-line mode;
     4. Installation is validated by default set of smoke tests.
-vars:
-  YUI_REST_API: 1
 schedule:
-  system_role:
-    - installation/system_role/select_role_minimal
+  default_systemd_target:
+    - installation/installation_settings/validate_default_target
```
Comment (jknphy) [inline schedule/yam/test_cases/textmode_installation_minimal_role.yaml:11]: you cannot remove minimal role:
https://openqa.suse.de/tests/13084484#step/select_role_minimal/1
(there is a visual bug in the libyui-rest-api that both look selected, but you can see later that is not the case)

Comment (tinawang123) [reply schedule/yam/test_cases/textmode_installation_minimal_role.yaml:13]: Thanks for your remind,  Update the file.

**Review by jknphy — APPROVED** [review]

**Pull request merged**